config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -1000,7 +1038,7 @@ periodics:
       - '--node-test-args=--container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/usr/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd\"]}"'
       - --node-tests=true
       - --provider=gce
-      - --test_args=--nodes=8 --focus="\[NodeConformance\]|\[NodeFeature:.+\]|\[NodeFeature\]" --skip="\[Flaky\]|\[Slow\]|\[Serial\]"
+      - --test_args=--nodes=8 --focus="\[NodeConformance\]" --skip="\[Flaky\]|\[Slow\]|\[Serial\]"
```

Comment:
If we plan to change this job to only be conformance, let's update the job name so it's clear. i.e. change the job name from `ci-cos-cgroupv2-containerd-node-e2e` to `ci-cos-cgroupv2-containerd-node-conformance` and test grid tab name from `cos-cgroupv2-containerd-node-e2e` to `cos-cgroupv2-containerd-node-conformance`.

Reply:
We should only do that if we also update all the other conformance-only jobs that are currently named the same way. Which is probably more churn than we want to do in one go?

Reply:
ah, gotcha, let's keep the name as existing then to be consistent  

though IMO as future improvement, I think it would be more clear if all the jobs would have the name `conformance` in them :)

Reply:
Definitely agree on that one 😅